package.json

```diff
@@ -2,29 +2,25 @@
   "name": "rally-web-platform",
   "version": "0.1.3",
   "scripts": {
-    "pre-build": "bash scripts/reset-build-directories.sh && bash scripts/add-protocol.sh && node scripts/add-version-object.js",
-    "dev": "npm run config:web:demo && npm run build:functions && firebase emulators:exec --project demo-rally --only auth,functions,firestore --ui 'npm run load:data && (npm run watch:functions & npm run watch:web)'",
+    "pre-build": "bash scripts/reset-build-directories.sh && bash scripts/add-protocol.sh && node scripts/add-version-object.js && ln -fs node_modules/@mozilla/rally-functions functions",
+    "dev": "npm run config:web:demo && firebase emulators:exec --project demo-rally --only auth,functions,firestore --ui 'npm run load:data && npm run watch:web'",
```

Comment:
`npm run dev` doesn't work out of the box anymore. Either we should add pre-build inside run dev, or just add the link functions step directly.

Reply:
Had to symlink functions here. It works now. Thanks